www/~/%username/emails/modify.json.spt

```diff
@@ -46,7 +47,10 @@ if action in ('add-email', 'resend', 'start-verification'):
         raise Response(400)
 
     participant.start_email_verification(address, *packages)
-    msg = _("Check your inbox for a verification link.")
+    if show_address_in_message:
```

Comment:
> Yeah, it seemed to me that it was best to have one code path for both, since we want linking packages to emails to also verify the email. That is, we don't want to make the user go through two verification flows (one for the address itself, a second for the packages) when from their point of view one should be sufficient. Am I thinking about that right?

I see how this makes sense. In that case, do we absolutely *need* to re-verify emails when claiming packages? If a user has a verified email, can't we just allow them to claim the package directly?